membership-attribute-service/app/models/AccessClaims.scala

```diff
@@ -0,0 +1,45 @@
+package models
+
+import com.gu.identity.auth._
+import com.gu.identity.model.User
+
+/** Claims that are used to determine which resources the user is authorised to access.Ch
+  *
+  * @param primaryEmailAddress
+  *   primary email address
+  * @param identityId
+  *   Identity ID
+  * @param username
+  *   User name, a synonym for display name. Also used to determine if this is a test user.
+  * @param userEmailValidated
+  *   true iff the user has validated their email address
+  */
+case class AccessClaims(
```

Comment:
I'm not sure about calling this class `AccessClaims` it seems very confusing for a dev coming to this code without background knowledge of how Okta works. Also we have variables scattered around the codebase now like `val user: AccessClaims` which is non-idiomatic and distracting. Could we call this `User` now that we don't have a user class from `com.gu.identity.model` anymore?

Reply:
I'm not opposed to a rename but I think it's important to make it clear that this is data included within the token itself when it was created not fresh data coming from the server side. It might not be up to date depending on what happened since then

Reply:
Yeah I see what you mean, maybe we can call it something like `UserTokenData` or `UserFromToken`?

Reply:
`UserFromToken` is a good suggestion.  Let's go with that.